shared/components/EventDetails/index.scss

```diff
@@ -3,6 +3,7 @@
 
```

Comment:
In this files there is the following line:
```
    &-container {
      flex: 1;
```
is flex 1 needed on the container? I understood that a number property is only relevant to the flex-items within a flex container